tempodb/encoding/vparquet2/dedicated_columns.go

```diff
@@ -0,0 +1,193 @@
+package vparquet2
+
+import (
+	v1 "github.com/grafana/tempo/pkg/tempopb/common/v1"
+	"github.com/grafana/tempo/tempodb/backend"
+)
+
+const (
+	dedicatedColumnTypeString    = "string"
+	dedicatedColumnScopeResource = "resource"
+	dedicatedColumnScopeSpan     = "span"
+)
+
+var (
+	// Column paths for spare dedicated attribute columns
+	dedicatedResourceColumnsByType = map[string][]string{
+		dedicatedColumnTypeString: {
+			"rs.list.element.Resource.DedicatedAttributes.String01",
```

Comment:
The column paths are actually not needed yet, but will be needed later for the querying part. For now only the number of entries is relevant.